usaspending_api/routers/replicas.py

```diff
@@ -14,22 +14,32 @@
 """
 
 
-class ReadReplicaRouter(object):
+class ReadReplicaRouter:
+
+    usaspending_db_list = (DEFAULT_DB_ALIAS, "db_r1")
+    broker_db_list = ("data_broker",)
+
     def db_for_read(self, model, **hints):
-        # these are the only models we write to; to deal with replication lag just get them from the source db
+        """ For reads, choose a connection randomly.  FilterHash and DownloadJob are writable tables so always
+            read from source (default) to mitigation replication lag issues. """
         if model in [FilterHash, DownloadJob]:
             return DEFAULT_DB_ALIAS
-        return random.choice([DEFAULT_DB_ALIAS, "db_r1"])
+        return random.choice(self.usaspending_db_list)
 
     def db_for_write(self, model, **hints):
-        # write to source db only (bc read replicas)
+        """ Write to source (default) db only because replicas are read only. """
         return DEFAULT_DB_ALIAS
 
     def allow_relation(self, obj1, obj2, **hints):
-        db_list = (DEFAULT_DB_ALIAS, "db_r1")
-        if obj1._state.db in db_list and obj2._state.db in db_list:
-            return True
-        return None
```

Comment:
Since the original implementation returned `True` or `None` should the changes match that instead of returning `True` or `False`? Not sure why it returns None, but it looks intentional.

Reply:
Of course you're right.

Reply:
Actually, after further reading, you return None if your router has no opinion which allows subsequent routers to cast their vote.  I think in this case, we're pretty certain we do not want to allow relations to other databases.  That and we currently have no other routers.